mlflow/R/mlflow/R/python.R

```diff
@@ -19,6 +19,7 @@ get_python_bin <- function() {
 python_bin <- function() {
   if (is.null(.globals$python_bin)) {
     python <- get_python_bin()
```

Comment:
Emm, why `get_python_bin` does not return python executable file path ?

Reply:
It does, it's just the path looks quite strange with the mix of "\\" and "/" so I'm trying to make it just include "/"

Reply:
got it, I missed the "dirname".

Could you make the improvement into the `get_python_bin` function ?

Reply:
Sure.